addons/point_of_sale/models/pos_session.py

```diff
@@ -1225,7 +1227,7 @@ def _get_invoice_receivable_vals(self, amount, amount_converted):
         }
         return self._credit_amounts(partial_vals, amount, amount_converted)
 
-    def _get_sale_vals(self, key, amount, amount_converted):
+    def _get_sale_vals(self, key, amount, amount_converted, tax_amount):
```

Comment:
We should not change the signature of any function in stable, right? But maybe we can get away with something like this?
```suggestion
    def _get_sale_vals(self, key, amount, amount_converted, *args):
        tax_amount, = args
```

Reply:
Changing private method signature is not forbidden (but we should avoid it). Also the function is not used anywhere else in the codebase so it's not a big deal IMO.

Reply:
https://github.com/odoo/odoo/wiki/Contributing

Reply:
Understood.